src/abstracts/egamefactory.py

```diff
@@ -270,3 +270,102 @@ async def gather_players(self) -> int:
         self.logging.info(f"number of players {self._num_players}")
 
         return self._num_players
+
+    async def check_continue(self, min_players: int = 1):
+        """
+        Brings up a dialogue on the main channel to ask players whether they want to continue with
+        same players, poll for new players or stop playing completely.
+
+        :param min_players: the minimum number of players who need to join in order to continue.
+            optional, defaults to 1
+        """
+        ipl = InteractionPipeline(
+            ButtonInteraction(
```

Comment:
Buttons are not vote interactions -- they will terminate the timer as soon as they are clicked. Either use `ChoiceInteraction`, or if you really want the custom emojis, I can (or you can) add a 

```py
def set_emojis(self, emojis: list) -> None:
```

function to `ChoiceInteraction`